src/data/lovelace.ts

```diff
@@ -12,10 +12,36 @@ export interface LovelaceConfig {
   background?: string;
 }
 
-export type LovelaceResources = Array<{
+export interface LovelaceResource {
+  id: string;
   type: "css" | "js" | "module" | "html";
   url: string;
-}>;
+}
+
+export interface LovelaceResourcesMutableParams {
+  res_type: "css" | "js" | "module" | "html";
+  url: string;
+}
+
+export interface LovelaceDashboard {
```

Comment:
Split this in a StorageDashboard and a YamlDashboard and make `LovelaceDashboard = `YamlDasboard | StorageDashboard`